templates/base_index.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+
+<html class="js" lang="en" dir="ltr">
```

Comment:
what does the js class do? Should we be adding that if JS is enabled?

Reply:
Doesn't do anything. I will remove it from the boiler plate project as well.